libmscore/rendermidi.cpp

```diff
@@ -402,7 +402,10 @@ static void collectMeasureEvents(EventMap* events, Measure* m, Staff* staff, int
                               for (MidiCoreEvent event : nel->events) {
                                     event.setChannel(channel);
                                     NPlayEvent e(event);
-                                    events->insert(std::pair<int, NPlayEvent>(tick, e));
+                                    if (e.dataA() == CTRL_PROGRAM)
```

Comment:
What about the other type of events? Would it make sense to send them on tick in advance too?